package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@toyota-research-institute/lakefront",
   "description": "React UI Components",
-  "version": "4.0.0",
+  "version": "4.2.2",
```

Comment:
nit: This should be a major update I believe (e.g 5.X), both here and in `package-lock.json`